doc/modules/mod_global_distrib.md

```diff
@@ -121,6 +121,7 @@ Global distribution modules expose several per-datacenter metrics that can be us
 * **endpoints** (list, default: `[{LocalHost, 5555}]`): A list of `{Host, Port}` tuples on which the server will listen for connections.
   `Host` can be given as a hostname, in which case it will be resolved to an IP address before first on module start.
   The endpoint list will be shared with other datacenters via the replicated backend.
+* **advertised_endpoints** (list | false, default: false): A list of `{Host, Port}` tuples which will be advertised in Redis and therefore used to establish connection with this node by other nodes. If not specified, `endpoints` value (after resolution) is considered `advertised_endpoints`. The host may be either IP or domain, just like in case of endpoints. The difference is, the domain name won't be resolved but inserted directly to the mappings backend instead.
```

Comment:
This would go to a separate config almost immediately.
I would probably have an ejabberd_local.cfg, defining a macro with adverised_endpoints value and than this local.cfg is included from the main ejabberd.cfg.
And than we use a macro from ejabberd.cfg.

(a bit offtopic, but can be a good practice to reduce number of configs)

Reply:
resolved, can go into best practices guide, if we would have one.